### DISCUSSION TITLE
When you update argo events chart with new version of argo-events

### DISCUSSION THREAD

**Discussion by AndriySidliarskiy** [argoproj/argo-helm#1469] [Q&A] [answered]
When you update argo events chart with new version of argo-events?

**Comment by jmeridth** [comment, marked as answer]
Underway [here](https://github.com/argoproj/argo-helm/pull/1465).  Once tests pass and get 👍 from reviewer, we will merge.  Cheers.

**Reply by jmeridth** [reply]
This has been merged.  Release shortly.

**Reply by pdrastil** [reply]
Done
